README.md

```diff
@@ -1,20 +1,37 @@
-# Tezos Ledger Applications
+# Tezos Baking
 
-## Investigation
+## Introduction
 
 
 ## Overview
 
-This repository contains Ledger application to support baking on Tezos blockchain:
-
-1. The "Tezos Baking" application (Nano S only) is for baking: signing new blocks and
-attestations. For more information about baking, see
+This repository contains Ledger application to support baking on Tezos blockchain , i.e. signing new blocks and pre-attestations and attestations. For more information about baking, see
 *[Benefits and Risks of Home Baking](https://medium.com/@tezos_91823/benefits-and-risks-of-home-baking-a631c9ca745)*.
 
 It is possible to do all of these things without a hardware wallet, but using a
 hardware wallet provides you better security against key theft.
 
+## Usage
+
+Install `tezos-baking` app from [Ledger-live](https://www.ledger.com/ledger-live). You may have to enable developer mode to see tezos-baking in the available apps list.
+You also will need to install `tezos-wallet` app for initial setup to stake tez for baking.
+
+### STEP 1
+  - Initialize your ledger device with your secret phrase and PIN and
+  - Install `tezos-baking` and `tezos-wallet` apps.
+
+### STEP 2
+  - Setup ledger for baking using instructions given in this file in section [Baking](./README.md#setup-tezos-baking-using-ledger) upto [Staking tez](./README.md#stake-tez-to-get-baking-rights).
+  - Exit the Tezos wallet app.
+### STEP 3
+  - `IMPORTANT` Change screensaver to 1 minute and `disable global PIN lock` in ledger settings.
```

Comment:
I'm not sure setting the screen saver duration to 1 minute is that important, we should just advise them not to turn it off.

Reply:
Right.